usagov/vote-gov-drupal
# VOTE- 3160 touch points mobile menu

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-3160

## Description

Add a button to the touch points form at the bottom of the mobile menu.

## Deployment and testing

### Post-deploy steps

1. lando retune
2. cd into theme, npm run build

### QA/Testing instructions

1. View the site in mobile dimensions and confirm the button matches the design.
2. Confirm the button displays for multiple screen sizes in mobile and tablet. Note the larger screen sizes place the button start aligned instead of centered.
3. Switch the Spanish and confirm the button no longer displays in the menu.
4. Switch to desktop and confirm the modal opens when the floating touch points button is clicked

## Checklist for the Developer




## Checklist for the Peer Reviewers
